src/tools/rust-analyzer/crates/rust-analyzer/tests/slow-tests/tidy.rs

```diff
@@ -142,6 +142,7 @@ fn check_cargo_toml(path: &Path, text: String) {
     }
 }
 
+#[cfg(not(feature = "in-rust-tree"))]
```

Comment:
fwiw it might be possible to use `--exclude check_merge_commits` or something like that. but no strong opinion, the cargo feature seems more extensible.

Reply:
> the cargo feature seems more extensible.

Yeah, I'm thinking of cfg-gating the "sysroot ABI" based on that feature.

Also, that test might die if using git subtree means `rust-lang/rust-analyzer` will also get merge commits in its tree, cf. [this zulip message](https://rust-lang.zulipchat.com/#narrow/stream/185405-t-compiler.2Frust-analyzer/topic/rust-analyzer.20as.20a.20subtree.3A.20experimental.20PR.20status/near/290113635).